radicle-cli/examples/workflow/2-cloning.md

```diff
@@ -0,0 +1,56 @@
+To create a local copy of a repository on the radicle network, we use the
+`clone` command, followed by the identifier or *RID* of the repository:
+
+```
+$ rad clone rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji
+✓ Tracking relationship established for rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji
+✓ Fetching rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji from z6MknSL…StBU8Vi..
+✓ Forking under z6Mkt67…v4N1tRk..
+✓ Creating checkout in ./heartwood..
+✓ Remote z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi created
+✓ Remote-tracking branch z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi/master created for z6MknSL…StBU8Vi
+
+🌱 Project successfully cloned under [..]/heartwood/
+
+```
+
+We can now have a look at the new working copy that was created from the cloned
+repository:
+
+```
+$ cd heartwood
+$ ls
+README
+$ cat README
+Hello World!
+```
+
+Let's check that the remote tracking branch was setup correctly:
+
+```
+$ git branch --remotes
+  rad/master
+  z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi/master
+```
+
+The first branch is ours, and the second points to the repository delegate.
+We can also take a look at the remotes:
+
+```
+$ git remote -v
+rad	rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (fetch)
+rad	rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (push)
+z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi	rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi (fetch)
+z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi	rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi (push)
+```
```

Comment:
This is the kind of thing that perhaps we can leave in the `rad-clone.md` test and not have it here, to keep the workflow examples a bit more focused?

Reply:
Showing how rad set's up git to interact with it helps the user build an intuition.  It goes along with 'git branch --remotes' and I recommend keeping it.